recipes/slurmpter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "slurmpter" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4dd1eeb5692ce2e92847fb01e73430323c912c3132d53418f0bd1f0ea5dcd34
+
+build:
```

Comment:
It looks like this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Yes the package is pure Python. I've added noarch to it.